mipdb/commands.py

```diff
@@ -141,9 +144,82 @@ def disable_dataset(dataset, schema, version):
 
 
 @entry.command()
+@cl.argument("name", required=True)
+@cl.option("-v", "--version", required=True, help="The schema version")
+@cl.option(
+    "-t",
+    "--tag",
+    default=None,
+    required=False,
+    help="A tag to be added/removed at the properties",
+)
+@cl.option(
+    "-kv",
+    "--key-value",
+    default=None,
+    nargs=2,
+    required=False,
+    help="A key value to be added/removed at the properties",
+)
+@cl.option(
+    "-a",
+    "--add",
+    is_flag=True,
+    required=False,
+    help="A flag that determines if the tag/key_value will be added",
+)
+@cl.option(
+    "-r",
+    "--remove",
+    is_flag=True,
+    required=False,
+    help="A flag that determines if the tag/key_value will be removed",
+)
```

Comment:
If the `remove` option is a flag then you don't need an `add` flag as well. You add something with no flag and you remove it with `--remove`.